docs/backup-restore.md

```diff
@@ -24,6 +24,34 @@ That means that you need to create a consistent snapshot by dumping all tables i
 For example, you can use `mysqldump` command with `--single-transaction` option in MySQL and `pg_dump` command in PostgreSQL to achieve that.
 Or when you use Amazon RDS (Relational Database Service) or Azure Database for MySQL/PostgreSQL, you can restore to any point within the backup retention period with the automated backup feature, which satisfies the requirement.
 
+## Cosmos DB
+
+You can create Cosmos DB account using `Continuous` backup policy to create continuous backups.
+
+The easiest way to take a transactionally-consistent backup for Scalar DB on Cosmos DB is to implement [scalar-admin](https://github.com/scalar-labs/scalar-admin) interface properly in your application, you can easily pause the application without losing ongoing transactions.
+You can use the middle value of the pause time as a restore point.
+
+You can restore the backup based on the [azure official guide](https://docs.microsoft.com/en-us/azure/cosmos-db/restore-account-continuous-backup#restore-account-portal) and you must change the default consistency to `STRONG` after restoring the data.
+
+## DynamoDB
+
+ScalarDB schema loader enables Point-in-time recovery (PITR) for each table in a DynamoDB while creating the schema. It will help you to create continuous backups.
+
+The easiest way to take a transactionally-consistent backup for Scalar DB on DynamoDB is to implement the [scalar-admin](https://github.com/scalar-labs/scalar-admin) interface properly in your application, you can easily pause the application without losing ongoing transactions.
+You can use the middle value of the pause time as a restore point.
+
+You can restore the tables one by one using the following steps from the Amazon DynamoDB console,
+
+    A. Restore the PITR (Point-in-time recovery) backup of tables except `scalardb.metadata` on the basis of [AWS official guide](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/PointInTimeRecovery.Tutorial.html#restoretabletopointintime_console).
+        
+    B. Create the backup of previously restored tables (A) using the [AWS official guide](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/Backup.Tutorial.html#backup_console).
```

Comment:
```suggestion
    B. Create the backup of previously restored tables (A) on the basis of [AWS official guide](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/Backup.Tutorial.html#backup_console).
```
I don't think `using a guide`  sounds good.

Reply:
What does `(A)` specify? The step `A`? (then it's pretty weird)

Reply:
Sorry for the inconvenience.
Document updated with modified steps
```
* Restore with PITR of table A to another table B (alias table)
* Take a backup of the restored table B (say, backup B)
* Remove table A
* Create a table named A with backup B
```